test/journeys/lib/test-helpers/index.js

```diff
@@ -36,3 +48,148 @@ export function moveMouse(aBrowser, selector) {
     aBrowser.moveToObject(selector);
   }
 }
+
+
+/**
+ * Creates a number of test users
+ * @param {number} count number of users to generate
+ * @param {Object} userInfo with config to pass to test user creator
+ * @returns {Object}
+ */
+export function createTestUsers(count, userInfo = {
+  docbrown: {displayName: 'Emmett Brown'},
+  lorraine: {displayName: 'Lorraine Baines'},
+  marty: {displayName: 'Marty McFly'},
+  biff: {displayName: 'Biff Tannen'}
+}) {
+  const users = {};
+  const userKeys = Object.keys(userInfo);
+
+  function storeUser(u, key) {
+    [users[key]] = u;
+    users[key].spark = new CiscoSpark({
+      credentials: {
+        authorization: users[key].token
+      },
+      config: {
+        logger: {
+          level: 'error'
+        }
+      }
+    });
+    return users[key];
+  }
+
+  for (let i = 0; i < count; i += 1) {
```

Comment:
If `count` is greater than `userKeys`, does it handle properly with a generic username from the test user library?

Reply:
Yup